src/main/kotlin/sliv/tool/common/Constants.kt

```diff
@@ -0,0 +1,7 @@
+package sliv.tool.common
+
+const val DEFAULT_COLOR_RED = 58
```

Comment:
bad idea to gather all project constants in single file, put them in the domain object